src/components/ui/Navbar.vue

```diff
@@ -67,6 +70,22 @@ import "vue-awesome/icons/pencil-ruler";
 export default {
   name: "Navbar",
   props: ["gameNav"],
+  data() {
+    return {
+      lastText: null,
+    };
+  },
+  methods: {
+    handleClick() {
+      this.$store.state.audio.playEffect("ui/pop");
+    },
+    handleHover(e) {
+      const text = e.target.innerText;
+      if (!text || text == this.lastText) return;
```

Comment:
```suggestion
      if (!text || text === this.lastText) return;
```

You probably should configure the linter to forbid comparisons with `==` as there are way too many pitfalls.